packages/@core.keystone/test.utils.js

```diff
@@ -241,19 +242,34 @@ const makeLoggedInClient = async (args) => {
             password: DEFAULT_TEST_USER_SECRET,
         }
     }
-    if (!args.email && !args.password) throw new Error('no credentials')
+    if (!(args.email || args.phone) && !args.password) throw new Error('no credentials')
     const client = await makeClient()
-    const { data, errors } = await client.mutate(SIGNIN_MUTATION, {
-        identity: args.email,
-        secret: args.password,
-    })
-    if (errors && errors.length > 0) {
-        throw new Error(errors[0].message)
-    }
-    client.user = {
-        email: args.email,
-        password: args.password,
-        id: data.auth.user.id,
+    if (args.email) {
+        const {data, errors} = await client.mutate(SIGNIN_MUTATION, {
```

Comment:
Maybe to rename it, like so:
```suggestion
        const {data, errors} = await client.mutate(SIGNIN_BY_EMAIL_MUTATION, {
```

Reply:
Renamed